docs/README.md

```diff
@@ -123,3 +130,420 @@ something like
+Let's address that in the next section.
+
+#### 7. Testing
+
+Let's get started testing the project. We will perform three actions:
+
+- Add a couple of posts
+- Retrieve all posts
+- Retrieve a specific post
+
+Booster applications provide you with a GraphQL API out of the box. Commands are mapped
+to mutations and read models to queries. To perform calls to the GraphQL API, you can use
+any HTTP client you want; we recommend you to use [Postwoman](https://postwoman.io/graphql).
+
+##### 7.1 Creating posts
+
+Use your favorite GraphQL client to run these mutations. No authorization header is
+required since we have allowed `all` to execute our commands and query the read models.
+
+```graphql
+mutation {
+  CreatePost(
+    input: {
+      postId: "95ddb544-4a60-439f-a0e4-c57e806f2f6e"
+      title: "Build a blog in 10 minutes with Booster"
+      content: "I am so excited to write my first post"
+      author: "Boosted developer"
+    }
+  )
+}
+```
+
+```graphql
+mutation {
+  CreatePost(
+    input: {
+      postId: "05670e55-fd31-490e-b585-3a0096db0412"
+      title: "Booster framework rocks"
+      content: "I am so excited for writting the second post"
+      author: "Another boosted developer"
+    }
+  )
+}
+```
+
+The expected response for each of those requests should be:
+
+```json
+{
+  "data": {
+    "CreatePost": true
+  }
+}
+```
+
+Note:
+
+> The IDs are generated on the client-side. For this example, we are not validating ID
+> uniqueness, consider using [a UUID generator](https://www.uuidgenerator.net/version4)
+> when creating more Posts.
+
+##### 7.2 Retrieving all posts
+
+Let's perform a GraphQL `query` that will be hitting our `PostReadModel`:
+
+```graphql
+query {
+  PostReadModels {
+    id
+    title
+    author
+  }
+}
+```
+
+It should response something like:
+
+```json
+{
+  "data": {
+    "PostReadModels": [
+      {
+        "id": "05670e55-fd31-490e-b585-3a0096db0412",
+        "title": "Booster framework rocks",
+        "author": "Another boosted developer"
+      },
+      {
+        "id": "95ddb544-4a60-439f-a0e4-c57e806f2f6e",
+        "title": "Build a blog in 10 minutes with Booster",
+        "author": "Boosted developer"
+      }
+    ]
+  }
+}
+```
+
+##### 7.3 Retrieving specific post
+
+It is also possible to retrieve specific a `Post` by adding the `id` as input, e.g.:
+
+```graphql
+query {
+  PostReadModel(id: "95ddb544-4a60-439f-a0e4-c57e806f2f6e") {
+    id
+    title
+    author
+  }
+}
+```
+
+You should get a response similar to this:
+
+```json
+{
+  "data": {
+    "PostReadModel": {
+      "id": "95ddb544-4a60-439f-a0e4-c57e806f2f6e",
+      "title": "Build a blog in 10 minutes with Booster",
+      "author": "Boosted developer"
+    }
+  }
+}
+```
+
+#### 8. Removing stack
+
+Now, let's destroy all the infrastructure created for running the application in the cloud.
+Execute the following command from the root of the project. For security reasons, you have
```

Comment:
I think that security implies other more dramatic meanings, like avoiding data leaks or something like that, safety sounds softer to me...

```suggestion
Execute the following command from the root of the project. For safety reasons, you have
```